### DISCUSSION TITLE
Payments declined by card issuers

### DISCUSSION THREAD

**Discussion by superstar1205** [Web-development-s/Stripe-with-laravel#47] [Q&A] [answered]
When a charge is submitted to the issuer of your customer’s card, they have automated systems and models that determine whether or not to authorize it. These systems analyze various signals, such as your customer’s spending habits, account balance, and card information such as the expiration date, address information and CVC.

**Comment by VitaminC1211** [comment]
Yes, I have solved it with googling easily, you want to share now?

**Comment by Phil-stack** [comment]
你好，已收到您的来信——马彬彬

**Reply by superstar1205** [reply]
Who are you? Can you share your skype name or telegram?

**Comment by superstar1205** [comment]
What are you doing now?

**Comment by agiledev0115** [comment]
Over all answers was wrong!!!

**Comment by accompany1205** [comment]
thanks once more

**Comment by crazydev0206** [comment]
Thanks for everything.
It works on my side, do you need my help?

**Comment by itechchef** [comment]
Hello

**Comment by VitaminC1211** [comment]
Who are you and how did you write me?

**Comment by dev0115** [comment, marked as answer]
ok
